qt/widgets/common/src/FitPropertyBrowser.cpp

```diff
@@ -3504,6 +3508,13 @@ void FitPropertyBrowser::addAllowedSpectra(const QString &wsName,
   }
 }
 
+void FitPropertyBrowser::removeWorkspaceAndSpectra(const std::string &wsName) {
+  removeWorkspace(wsName);
```

Comment:
I think that might be the plan - unfortunately the test coverage on this file is very poor and I didn't want to make any changes to the FitPropertyBroswer this late in the release. If you agree, I plan to make another issue for this - and to add an observed so that the in normal circumstances FitPropertyBrowsers can handle workspaces being removed etc.